### PR TITLE
Custom card set with average

### PR DESCRIPTION
**Pull request by royds4**
Added a custom card set to vote from 1 -5 and calculate the average when using it

### CODE REVIEW

**Comment by amaury-biquet** [comment]
Those cards from 1 to 5 are very useful for the confidence vote we perform each sprint.

**Comment by amaury-biquet** [comment]
@cschleiden can you help us with the review and merge please ?

File: src/pages/session/components/workItemView.tsx [hunk 1]
```diff
@@ -142,6 +143,22 @@ class WorkItemView extends React.Component<IWorkItemProps & typeof Actions> {
                                                     );
                                                 })}
                                             </div>
+                                            {confidence && (
```
Comment (cschleiden) [inline src/pages/session/components/workItemView.tsx]: I'm thinking of showing the average for all Numeric cards, there have been many asks for it. What do you think?

Comment (amaury-biquet) [reply src/pages/session/components/workItemView.tsx]: seems good, what do you think @royds4 ?

Comment (royds4) [reply src/pages/session/components/workItemView.tsx]: @cschleiden @amaury-biquet Agreed. Already updated.

File: src/pages/session/components/workItemView.tsx [hunk 2]
```diff
@@ -232,6 +249,7 @@ export default connect(
             selectedWorkItem: session.selectedWorkItem!,
             estimates,
             revealed: session.revealed,
+            confidence: session.cardSet!.type === CardSetType.Numeric,
```
Comment (cschleiden) [inline src/pages/session/components/workItemView.tsx]: just `showAverage`?

Comment (royds4) [reply src/pages/session/components/workItemView.tsx]: Ok

**Comment by amaury-biquet** [comment]
@cschleiden , Thanks a lot for merging our proposed change.
Would you be able to publish a new release for this ? 2.2.4 ?

**Comment by JonathanAsbury-ACS** [comment]
@cschleiden
I'm willing to take over ownership of this solution and carry it forward should there be interest.

**Comment by liliankasem** [comment]
My team and I are also interested in this feature, any updates on a new release to get this feature in? :D

**Pull request merged**